Apps/Database/src/Assets/docs/EmailValidationTemplate.html

```diff
@@ -21,7 +21,7 @@
                     scope="col"
                     style="text-align: left; width:350px;"
                 >
-                    <div role="img" aria - label="Health Gateway Logo">
```

Comment:
Attribute should be `aria-label`, not `arialabel`.

Reply:
It is 'aria-label' isn't it?

<img width="1053" alt="Screenshot 2024-05-03 at 12 09 34 PM" src="https://github.com/bcgov/healthgateway/assets/58790456/e4e97f1a-ec07-4ae6-83fc-8d6703017da1">

Reply:
Yes, you're right.  Somehow I missed seeing the corrected line.